samples/requesterPays.js

```diff
@@ -0,0 +1,217 @@
+/**
+ * Copyright 2017, Google, Inc.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * This application demonstrates how to perform basic operations on buckets with
+ * the Google Cloud Storage API.
+ *
+ * For more information, see the README.md under /storage and the documentation
+ * at https://cloud.google.com/storage/docs.
+ */
+
+'use strict';
+
+function enableRequesterPays(bucketName) {
+  // [START storage_enable_requester_pays]
+  // Imports the Google Cloud client library
+  const Storage = require(`@google-cloud/storage`);
+
+  // The name of the bucket to enable requester-paying for, e.g. "my-bucket"
+  // const bucketName = "my-bucket";
```

Comment:
Supposed to be commented?

Reply:
Yes. If you execute the `requesterPays.js` file, then `bucketName` is passed into the `enableRequesterPays` function on line 26, ala `node requesterPays.js enable "some-bucket-name"`.

Someone looking at the sample on [cloud.google.com](https://cloud.google.com) won't see the wrapper function, and even if they did, just seeing a `bucketName` argument doesn't tell them very much. So, for those looking at the sample in the docs, who might copy and paste the sample (without the wrapper function) into their app, all they have to do us uncomment the commented variable initializations to make it work. Those comments also serve to give users examples of real values.

It's a larger conversation whether the samples should be useful command line programs. If we were to do away with that, then we could do away with the wrapper functions and just hard-code the variable values. That would, however, make the samples harder to test. The samples' tests in their current form double as system tests for the client library.